code/modules/projectiles/guns/projectile/automatic/slaught_o_matic.dm

```diff
@@ -14,6 +14,7 @@
 	load_method = MAGAZINE
 	mag_well = MAG_WELL_SMG
 	auto_eject = FALSE
+	serial_type = "FS"
```

Comment:
I'm pretty sure idea of these guns is to be untraceable but we don't have "There's no serial number whatsoever" so ehhhhh